src/components/ModalDialog/ModalDialog.css

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2022 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+.container {
+  position: relative;
+  margin: 0;
+  padding: 0;
+  height: var(--ModalDialog-Height, auto);
+  min-height: 200px;
+  max-height: min(var(--ModalDialog-Height, 680px), calc(100vh - var(--spacing-huge)));
+  max-width: calc(100vw - var(--spacing-huge));
+  display: grid;
+  grid-template-rows: auto auto 1fr auto;
+  grid-template-areas:
```

Comment:
nice

Reply:
I flip'n love grid. Can't wait for subgrid to be viable!